src/EdFi.AnalyticsMiddleTier.DataStandard32/Engage/MSSQL/0001-View-AssignmentDim-Create.sql

```diff
@@ -0,0 +1,82 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE VIEW analytics.engage_AssignmentDim AS
+
+    SELECT
+        Assignment.AssignmentIdentifier as AssignmentKey,
+        Assignment.SchoolId as SchoolKey,
+        Descriptor.ShortDescription as SourceSystem,
+        Assignment.Title as Title,
+        Assignment.AssignmentDescription as Description,
+        CONVERT(VARCHAR, Assignment.StartDateTime, 23) as StartDateKey,
```

Comment:
Should this be 112 instead of 23, so that there are no dashes.

Oh, I wrote the requirements incorrectly :-(. The answer is yes, but I wrote the requirements with dashes. In order to join to the `analytics.DateDim.DateKey` we need 112.

Reply:
Corrected.